release_dashboard/lib/services/local_conductor.dart

```diff
@@ -0,0 +1,79 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of String source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:io' as io;
+
+import 'package:conductor_core/conductor_core.dart'
+    show Checkouts, StartContext, Stdio, VerboseStdio, defaultStateFilePath, readStateFromFile;
+import 'package:conductor_core/proto.dart' as pb;
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:platform/platform.dart';
+import 'package:process/process.dart';
+
+import 'conductor.dart';
+
+/// Service class for using the conductor in a local environment.
+///
+/// This is the production version of the conductor, only intended for releases.
+class LocalConductorService extends ConductorService {
+  final FileSystem fs = LocalFileSystem();
+  final Platform platform = LocalPlatform();
+  final ProcessManager processManager = LocalProcessManager();
+  final Stdio stdio = VerboseStdio(
+    stdout: io.stdout,
+    stderr: io.stderr,
+    stdin: io.stdin,
+  );
+
+  Directory get rootDirectory => fs.directory(String.fromEnvironment('HOME'));
```

Comment:
Does this work? I tried:

```
void main() {
  const String foo = String.fromEnvironment('FOO');
  print("FOO = \"$foo\"");
}
```

And then ran it with:

```
FOO=bar dart ./foo.dart
FOO = ""
```

Reply:
It works on my linux machine - 
![image](https://user-images.githubusercontent.com/2148558/139495073-475e5539-813a-4888-ad13-85f92e12c74d.png)